src/dotnet/commands/dotnet-build3/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.DotNet.Cli
+{
+    public class Build3Command
```

Comment:
The eventual verb will continue to be `build`. We didn't want to introduce `msbuild` as a concept here since folks are likely to assume it will stick around. `dotnet build` is where we're headed.